.github/workflows/sanity.yml

```diff
@@ -3,7 +3,7 @@ on:
   pull_request:
   push:
     branches:
-      - rolling
+      - iron
```

Comment:
This seems like an unrelated change? if i am not mistaken...

i thought that we need to change `runs-on: ubuntu-24.04` for `rolling` branch?

Reply:
This PR is against `iron`. The change on this line should hopefully make CI run against new commits on the `iron` branch.

Reply:
yep, but I rarely backport things/change things in older distros, so it's not always the case.